ProxyAppsSuite/tests/ember.yaml

```diff
@@ -0,0 +1,470 @@
+_base:
+    summary: Large scale modeling of high-performance interconnects
```

Comment:
Does it model without the SST simulator to ingest the results?  may change this to mpi and shmem performance tests.